yo/api_methods.py

```diff
@@ -1,12 +1,27 @@
 # coding=utf-8
 import logging
 
+from yo import transports
+
+from yo import storage
+from yo.storage import users
+
 from jsonrpcserver.aio import methods
 
 logger = logging.getLogger(__name__)
 
+def add_api_method(func):
+    """ Used to add the yo. prefix
+    """
+    methods.add(func,name='yo.%s' % func.__name__)
+
+# test method
+@add_api_method
+async def test(db=None):
```

Comment:
I plan to do precisely this - in fact, I have a plan to build a general template for all these webservice apps that will include this info.

The reason to add the test method here at first was just for my own purposes while getting used to the code and verifying (without messing with other methods) that it was all running.